nvdaHelper/remote/apiHook.cpp

```diff
@@ -104,9 +104,9 @@ void* apiHook_hookFunction(const char* moduleName, const char* functionName, voi
 	}
 	LOG_DEBUG("requesting to hook function " << functionName << " at address 0X" << std::hex << realFunc << " in module " << moduleName << " at address 0X" << moduleHandle << " with  new function at address 0X" << newHookProc);
 	void* origFunc;
-	int res;
+	MH_STATUS res;
 	if((res=MH_CreateHook_fp(realFunc,newHookProc,&origFunc))!=MH_OK) {

```

Comment:
assignments done inside `if` blocks can easily become a source of error. I would much prefer to see this changed to:
```
MH_STATUS res = MH_CreateHook_fp(realFunc, newHookProc, &origFunc);
if( res != MH_OK) {
```
This gets rid of the declared but not initialised `res`, and reduces the logic inside the if statement.